lib/controller.ts

```diff
@@ -205,6 +228,10 @@ export class Controller {
             logger.error('Failed to stop Zigbee2MQTT');
             await this.exit(1, restart);
         }
+
+        if (sdNotify) {
+            sdNotify.stopWatchdogMode();
```

Comment:
```suggestion
            sdNotify?.stopWatchdogMode();
```

The `if` for this and the other cases can be removed when using this syntax.